.github/workflows/ca-clone-sequential-test.yml

```diff
@@ -1429,6 +1429,774 @@ jobs:
 
           diff expected actual
 
+      ####################################################################################################
+      # Switch cert request ID generator to newLegacy and verify if serials
+      # have gaps when range is updated
+      #
+      # It should work like the legacy but with correct range.
+      - name: Switch primary to newLegacy
+        run: |
+          # Update CS.cfg
+          docker exec primaryds ldapsearch \
+              -H ldap://primaryds.example.com:3389 \
+              -D "cn=Directory Manager" \
+              -w Secret.123 \
+              -b ou=certificateRepository,ou=ranges,dc=ca,dc=pki,dc=example,dc=com \
+              -s one \
+              -o ldif_wrap=no \
+              -LLL '(&(host=primary.example.com)(SecurePort=8443))' beginRange | tee output
+
+          # Since there is a range the value are retrieved from the range. In general, id range is not
+          # present the value from CS.cfg are used.
+          BEGIN_SERIAL=0x$(cat output | sed -n 's/beginRange: \(.*\)/\1/p' | sort -n | tail -1)
+          END_SERIAL=$(printf "0x%x\n" $(( BEGIN_SERIAL + 0x12 -1 )) )
```

Comment:
We probably cannot not assume that the range size will be a constant. It looks like the ranges are smaller for clones.

Reply:
As shown by the test I just updated, the size does change under certain circumstances.

Reply:
The problem is not for the clones. The first ranges are bigger and after the size become constant. It is evident also in the single CA sequential. This is because of the initial error on the `nextRange`.

Reply:
The clone does get a small portion of the main server's range as defined in `dbs.requestCloneTransferNumber` (i.e. 5) and it will maintain that size when allocating new ranges, so it's not the full range as defined in `dbs.requestIncrement` (i.e. 10), at least in SSNv1.

Reply:
The value for the transferred range are in hex and they do not generate any activity in the DS so it should not be a problem because it is the case of no range entry and the value in configuration is used